portality/api/v1/data_objects/article.py

```diff
@@ -48,14 +48,6 @@
                     }
                 },
 
-                "link": {
```

Comment:
I think this should still be in, like:

```
"link": {
    "fields": {
        "type": {"coerce": "link_type"},
        "url": {"coerce": "unicode"},
        "content_type": {"coerce": "link_content_type"}
    }
},
```
Note `url` has been replaced by `unicode`. This is a more minimal change to the struct to allow bad URLs out of the system. I don't want to go too far down the route of making this model permissive.

In general, if we have a big problem with bad data causing 500 errors on outgoing articles, we should catch that exception when we are enforcing the outgoingarticleDO and give a JSON response saying we can't give it out.

Reply:
Agreed.  I pondered this same point for a while and went the other way, but if @Steven-Eardley thinks it should be in I'd +1 that.